docker-compose.yml

```diff
@@ -136,41 +174,43 @@ services:
       api:
         condition: service_healthy
     command: ['./scripts/start.worker.sh']
+    environment:
+      <<: *common-environment
+      LAGO_REDIS_CACHE_URL: "redis://${LAGO_REDIS_CACHE_HOST}:${LAGO_REDIS_CACHE_PORT}"
```

Comment:
Redis is used by each backend services, maybe we can move them to `common`  also?

Reply:
Indeed, that’s also what I did initially. The problem is that interpolation doesn’t work when placed inside variables. It remains in the form "redis://${LAGO_REDIS_CACHE_HOST}:${LAGO_REDIS_CACHE_PORT}" without substituting the variables. :/